draganddrop/src/main/java/com/microsoft/device/display/samples/draganddrop/utils/DragData.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.draganddrop.utils
+
+class DragData(
+    type: MimeType = MimeType.TEXT_PLAIN,
+    data: Any? = null
+) {
+    val type = type
+    val data = data
```

Comment:
another small thing but I think you can just declare the properties as vals in the constructor and then the class body can be empty